main.py

```diff
@@ -4,7 +4,9 @@
 
 from topo_processor.metadata import DataType, create_collection
 
-data_folder = os.path.join(os.getcwd(), "test_data")
+data_folder = os.path.join(os.getcwd(), "test_data", "tiffs")
```

Comment:
These changes related to the test_data directory belong in a different pull request.